src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -195,6 +195,18 @@
   <data name="DoNotUseCountWhenAnyCanBeUsedTitle" xml:space="preserve">
     <value>Do not use Count() or LongCount() when Any() can be used</value>
   </data>
+  <data name="PreferConvertToHexStringOverBitConverterTitle" xml:space="preserve">
+    <value>Prefer 'Convert.ToHexString' and 'Convert.ToHexStringLower' over call chains based on 'BitConverter.ToString'</value>
+  </data>
+  <data name="PreferConvertToHexStringOverBitConverterDescription" xml:space="preserve">
+    <value>Use 'Convert.ToHexString' or 'Convert.ToHexStringLower' when encoding bytes to a hexadecimal string representation, as they are more efficient and allocation-friendly than using 'BitConverter.ToString' in combination with 'string.Replace' to replace dashes and 'string.ToLower'.</value>
```

Comment:
```suggestion
    <value>Use 'Convert.ToHexString' or 'Convert.ToHexStringLower' when encoding bytes to a hexadecimal string representation. These methods are more efficient and allocation-friendly than using 'BitConverter.ToString' in combination with 'string.Replace' to replace dashes and 'string.ToLower'.</value>
```

Reply:
Is this analyzer C#-specific? Otherwise I think we should use capitalized "String" type in the message.

Reply:
It is not, I've applied your suggestions in 309d4eae067e290180ab3feeef9a36cee0d240f2, thanks for the feedback! 😃